docs/_articles/en/troubleshooting.md

```diff
@@ -93,3 +93,10 @@ being ignored by the scanner, since dependent modules may have TypeScript files
 Make sure that the entries for eslint and eslint-typescript in `${HOME}/.sfdx-scanner/Config.json` both exclude the folder
 used by your framework. For example, to exclude Yarn/NPM dependencies, add `!**/node_modules/**` to the `targetPatterns`
 property, and add `!**/bower_components/**` to exclude Bower dependencies.
+
```

Comment:
This is included with https://github.com/forcedotcom/sfdx-scanner/pull/203 @rmohan20 opened 203 yesterday, but it hasn't been merged yet.